lib/matplotlib/animation.py

```diff
@@ -1543,7 +1543,6 @@ def __init__(self, fig, func, frames=None, init_func=None, fargs=None,
         # Amount of framedata to keep around for saving movies. This is only
         # used if we don't know how many frames there will be: in the case
         # of no generator or in the case of a callable.
-        self.save_count = save_count
```

Comment:
Might be easiest just to leave this line, and then do the forced conversion where you do it below.